apps/system/js/app_window.js

```diff
@@ -1026,6 +1026,7 @@
       }
       // Send event instead of call crash reporter directly.
       this.publish('crashed');
+      this.hideContextMenu();
```

Comment:
Does this code is called when the app is killed from the card view ? If no, you may want to move that to destroy() instead.